src/themes/default/components/core/blocks/Microcart/Product.vue

```diff
@@ -204,6 +217,16 @@ export default {
     },
     productQty () {
       return this.editMode ? this.getEditingQty : this.product.qty
+    },
+    isSimpleOrConfigurable () {
+      if (
+        this.product.type_id === 'simple' ||
+        this.product.type_id === 'configurable'
+      ) { return true }
+      return false
+    },
+    isUpdateCartDisabled () {
+      return this.quantityError || this.isStockInfoLoading || (this.isOnline && (!this.maxQuantity && this.isSimpleOrConfigurable))
```

Comment:
```suggestion
      return this.quantityError || this.isStockInfoLoading || (this.isOnline && !this.maxQuantity && this.isSimpleOrConfigurable)
```

Reply:
and even you can split this condition as it's a but too long

Reply:
done